docs/docs/api/index.md

```diff
@@ -0,0 +1,117 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Automate tasks with our API
+
+Our API is designed to be simple and easy to use. It is based on RESTful
+principles and uses standard HTTP methods. This means that you can use the API
+to automate tasks such as creating and managing projects, sessions, and users.
+
+<!-- prettier-ignore -->
+!!! warning
+    We try to keep the API as stable as possible, but we cannot guarantee that
+    it will not change in the future. If you are using the API, make sure to
+    check our [release notes](https://github.com/DSD-DBS/capella-collab-manager/releases)
+    for any breaking changes.
+
+## API Documentation
+
+Please use these links to access the API documentation:
+
+- [:octicons-link-external-16: SwaggerUI](swagger.md) (Interactive SwaggerUI
+  documentation)
+- [:octicons-link-external-16: ReDoc](redoc.md) (Interactive ReDoc
+  documentation)
+- [:octicons-link-external-16: OpenAPI Specification, formerly Swagger Specification](openapi.md)
+  (JSON)
+
+<!-- prettier-ignore -->
+!!! note
+    The API documentation is only available via the documentation
```

Comment:
I would just bold part or all of this sentence and delete the entire "It is not available on" section.